java/src/client/managers/UserManager.java

```diff
@@ -18,8 +18,13 @@ public UserManager(){
      * @pre
      * @post returns if the user was successfully created or not
      */
+<<<<<<< HEAD
```

Comment:
Not really sure how UserManager got merge conflicts. It doesn't look like you changed anything...